packages/cli-kit/src/private/node/ui/components/SelectInput.tsx

```diff
@@ -251,7 +259,7 @@ function SelectInputInner<T>(
     )
   } else {
     const optionsHeight = initialItems.length + maximumLinesLostToGroups(initialItems)
-    const maxKeyLength = initialItems
+    const maxKeyLength = itemsWithKeys
```

Comment:
I disagree with this change. We should use `initialItems` so the width doesn't flicker as items are scrolled through.